### DISCUSSION TITLE
Inter Organizations access to the repositories

### DISCUSSION THREAD

**Discussion by ghost** [community/community#64088] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

We have more than one Organization and when there is a lot of members it can be not so suitable to manage accesses.

It maybe very helpfully to have a way to
1. Add different Organization or just its Team as a member of the local Team
   `Local Team` = `@Different Org`
   `Local Team` = `@Different Org/Team`
2. Add different Organization permissions at the repository level
   `Repository access` = `@Different Org`
   `Repository access` = `@Different Org/Team`

Did I miss something and it is already possible?

**Comment by 0xChintan** [comment, marked as answer]
Yes, you are correct. It is not currently possible to add a different organization or team as a member of a local team, or to add different organization permissions at the repository level.
